src/main/java/com/example/bookstorewebapp/model/role/Role.java

```diff
@@ -0,0 +1,35 @@
+package com.example.bookstorewebapp.model.role;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+import org.springframework.security.core.GrantedAuthority;
+
+@Data
```

Comment:
If i'm not wrong all you need here are getters and setters, so maybe it would be better to use `@Getter` and `@Setter` instead of `@Data`?